<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHalaman">
    <property name="title">
     <string>Halaman</string>
    </property>
    <addaction name="actionUsers"/>
    <addaction name="actionOrganisasi"/>
    <addaction name="actionBantuan"/>
    <addaction name="actionKategoriBantuan"/>
    <addaction name="actionLaporan"/>
   </widget>
   <addaction name="menuHalaman"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionOrganisasi">
   <property name="text">
    <string>Organisasi</string>
   </property>
  </action>
  <action name="actionBantuan">
   <property name="text">
    <string>Bantuan</string>
   </property>
  </action>
  <action name="actionKategoriBantuan">
   <property name="text">
    <string>KategoriBantuan</string>
   </property>
  </action>
  <action name="actionLaporan">
   <property name="text">
    <string>Laporan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
